chrono/settings.py

```diff
@@ -47,14 +49,20 @@
     'django.contrib.sessions',
     'django.contrib.messages',
     'django.contrib.staticfiles',
+
+    'rest_framework',
+    'django_filters',
+
 ] + [
-    '{}.{}'.format(
+    '{}.{}.apps.{}Config'.format(
         APPS_DIR_NAME,
         app,
+        ''.join([word.title() for word in app.split('_')]),
     ) for app in LOCAL_APPS
```

Comment:
Shouldn't fstrings be preferred? Is it called fstrings @thenav56 ?

Reply:
empty expression not allowed in f-string @tnagorra

Reply:
**fstrings** is bit difficult to use with complex argument like `''.join([word.title() for word in app.split('_')])`.